### ISSUE TITLE
[Register a prescription] 

### ISSUE THREAD

**Issue by Benilde** [esaude/esaude-emr-poc#171]
**[Actual Results]:** The clinical can prescribe a medicine without doing a consultastion
**[Expected results]:** The system should only allows the clinical to prescribe any drug after the patient pass for the consultation.
**[Steps to reproduce]:** Clinic Module -> Patient Dashboard -> Prescriprion -> Create any prescription before go to the consultation.

**Comment by steliomo** [comment]
This is not a pharmacy bug. I'll move it POC board

**Comment by valvijo** [comment]
I don't think this is a bug! @Benilde what are the implications of doing prescription before or without consultation forms entry?

**Comment by Benilde** [comment]
@valvijo. This is one of the requirements that was proposed by MOH in one of the meetings that we had. They have said that they do not want to a patient receive a prescription without doing a consultation with the clinician.
